### PR TITLE
Make icons corresponding to links clickable

### PR DESCRIPTION
**Pull request by Victor-Chris**
Updated _index.html under main content to make the icons corresponding to the necessary links clickable.

### CODE REVIEW

File: website/content/en/_index.html [hunk 1]
```diff
@@ -22,20 +22,43 @@
...
-<a class="mr-3 mb-4 text-secondary" href="{{< relref "/docs/usage" >}}">
-Get started with Lima <i class="fas fa-arrow-alt-circle-right ml-2"></i></a>
-{{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-brands fa-github" title="Contributions welcome" %}}
-<a class="mr-3 mb-4 text-secondary" href="{{< relref "/community" >}}">
-Join the Lima community <i class="fas fa-arrow-alt-circle-right ml-2"></i></a>
-{{% /blocks/feature %}}
+
+  <div class="col-lg-4 mb-5 mb-lg-0 text-center">
```
Comment (balajiv113) [inline website/content/en/_index.html:26]: This has lot of duplicates now. 
Can you try adding url attribute to `{{% blocks/feature `. See if that becomes clickable 

Source https://www.docsy.dev/docs/adding-content/shortcodes/#blocksfeature

**Review by balajiv113 — CHANGES_REQUESTED** [review]

File: website/content/en/_index.html (hunk 1 above)
Comment (Victor-Chris) [reply website/content/en/_index.html:26]: I agree about the duplicates as well. I tried a number of options to do away with them, but with no success. I'm considering exploring the option of creating a custom block to minimise the duplication.

The option suggested with the link you've shared seems to just add a "Read more..." link to the **{{% blocks/feature** 
 without making the block/icons clickable.

Screenshot 2023-10-03 at 08 53 54

Comment (balajiv113) [reply website/content/en/_index.html:26]: Ah okay. In that case, do create a custom shortcode to minimise the duplication

We already have some custom defined shortcode here 
https://github.com/lima-vm/lima/blob/master/website/layouts/shortcodes/blocks

Comment (Victor-Chris) [reply website/content/en/_index.html:26]: Sure thing, I'd already taken a look through some of them. 
Let me try a custom shortcode and I'll revert afterwards.

**Comment by Victor-Chris** [comment]
Have you had a chance to check out the latest commit @balajiv113 ?

**Comment by balajiv113** [comment]
@Victor-Chris 
Kindly squash your commits and also fix the yamllint issue in the new yaml file

File: website/data/helpfullinks.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- link: /docs/installation
```
Comment (balajiv113) [inline website/data/helpfullinks.yaml:1]: Naming conventions in this file could be better. Maybe something like
```
header: Install Lima
icon: fa-solid fa-code
link: /docs/installation
label/linkLabel/linkText: Get Lima Here
```

File: website/layouts/shortcodes/blocks/helpfullinks.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{{ $usedBy := site.Data.helpfullinks -}}
+
+{{ range $usedBy -}}
+  <div class="col-lg-4 mb-5 mb-lg-0 text-center">
+    <a class="text-white" href="{{ .link }}">
```
Comment (balajiv113) [inline website/layouts/shortcodes/blocks/helpfullinks.html:5]:
```suggestion
    <a class="text-primary" href="{{ .link }}">
```

Comment (balajiv113) [reply website/layouts/shortcodes/blocks/helpfullinks.html:5]: This way, when we want to change the primary color, these won't be affected

**Review by balajiv113 — CHANGES_REQUESTED** [review]

**Review by balajiv113 — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**